src/components/panels/TimePanel.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+<b-col md="6">
+  <h3 class="text-center">{{headertext}}</h3>
+  <b-list-group>
+    <b-list-group-item :style="styleObject" class="list-group-item-borderless">
+      <span class="pull-left">
+        <b>Currently it is:</b>
+      </span>
+      <b-badge v-if="time.isDay" class="pull-right" style="margin-right: 5px" variant="success">Day</b-badge>
+      <b-badge v-else class="pull-right" style="margin-right: 5px" variant="dark">Night</b-badge>
+    </b-list-group-item>
+    <b-list-group-item :style="styleObject" class="list-group-item-borderless">
+      <span class="pull-left">
+        <b>Time until <span v-if="!time.isDay">Day</span><span v-else>Night</span>: </b>
+      </span>
+      <b-badge variant="info" :style="styleObject" class="pull-right" style="margin-right: 5px">{{time.timeLeft}}</b-badge>
+    </b-list-group-item>
+    <b-list-group-item :style="styleObject" class="list-group-item-borderless">
+      <span class="pull-left">
+        <b>Time at <span v-if="!time.isDay">Day</span><span v-else>Night</span>:</b>
+      </span>
+      <b-badge variant="info" :style="styleObject" class="pull-right" style="margin-right: 5px">{{earthtimezonetime}}</b-badge>
+    </b-list-group-item>
+    <b-list-group-item class="list-group-item-borderbottom">
+    </b-list-group-item>
+  </b-list-group>
+</b-col>
+</template>
+
+<script>
+  import moment from 'moment'
```

Comment:
use the time badge instead of importing moment again.

Reply:
i get a wired -22d   string when using  your  badage  for  the date

Reply:
it's probably sending milliseconds instead of seconds.

Reply:
its getting   the data from the api: expiry:"2018-10-25T00:08:00.000Z"